sdk/test-utils/recorder/CHANGELOG.md

```diff
@@ -2,18 +2,26 @@
 
 ## 1.0.0 (Unreleased)
 
+## 2021-05-12
+
+- Extended the `requestBodyTransformations` from #14897 to handle browser tests as well.
+  - Adds a default transformation to strip out the scope in the request body with the string `"https://sanitized/"`
```

Comment:
Instead of default transformation, add a default replacement for node now that the msal requests are mocked with #14993 and they don't care about the presence of scope.